compiler-core/src/language_server/tests/snapshots/gleam_core__language_server__tests__action__extract_variable.snap

```diff
@@ -11,6 +11,6 @@ pub fn main() {
 
 ----- AFTER ACTION
 pub fn main() {
-  let value = [1, 2, 3]
-  list.map(value, int.add(1, _))
+  let list = [1, 2, 3]
```

Comment:
I wonder if `ints` would be a good name for lists of ints, `floats` for floats, etc. What do you think?

Reply:
That would be really noce, maybe we could limit it to the core types we know will produce a good plural form and still fallback to `list` for user defined types?

Reply:
Could do, why would you want it to work that way?

Reply:
I think it wouldn't look nice if we just added an s at the end of the type name maybe generating gibberish
```
// from gleam's codebase, this would
// become variable_namess
type VariableNames
type Fish
type Dice
```

Reply:
Not adding if it ends in `s` would be fine I think.

Reply:
Ok!